kernel/papa/src/main.rs

```diff
@@ -25,6 +25,9 @@ extern crate h1;
 extern crate kernel;
 extern crate cortexm3;
 
+use crate::kernel::hil::gpio::Configure;
+use crate::kernel::hil::gpio::Output;
```

Comment:
These `use crate::kernel::...` statements don't look right to me. The `crate` keyword is used to import items from the current crate, not from other crates. I think these belong in the `use kernel::...` section below.

Also, then can be a single line:

```rust
use kernel::hil::gpio::{Configure, Output};
```

Reply:
Ooops, yeah! How did that happen? Good catch! Fixed.